src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceSetWrapper.cs

```diff
@@ -32,9 +34,25 @@ public ODataResourceSet ResourceSet
             }
         }
 
+        /// <summary>
+        /// Gets the wrapped <see cref="ODataResourceSetBase"/>.
+        /// </summary>
+        public ODataResourceSetBase ODataResourceSetBase
+        {
+            get
+            {
+                return Item as ODataResourceSetBase;             
+            }
+        }
+
         /// <summary>
         /// Gets the nested resources of this ResourceSet.
         /// </summary>
         public IList<ODataResourceWrapper> Resources { get; private set; }
+
+        /// <summary>
+        /// Delta Links and deleted links
+        /// </summary>
+        public IList<ODataDeltaLinkWrapper> DeltaLinks { get; private set; }
```

Comment:
>public IList<ODataDeltaLinkWrapper> DeltaLinks { get; private set; } [](start = 8, length = 68)

public IList<ODataDeltaLInkWrapper> DeltaLinks {
    get {
          return ResourceItems.OfType<ODataDeltaLinkWrapper>();
   }
}

Reply:
This was resolved by removing the DeltaLinks property (since we no longer try to process them separately), right?

---
In reply to: [495298437](https://github.com/OData/WebApi/pull/2275#discussion_r495298437) [](ancestors = 495298437)